R/replace_identified_by_x.R

```diff
@@ -0,0 +1,34 @@
+#' Replace each identified positions in protein by "x"
+#'
+#' Helper function for the calculation of sequence coverage, replaces identified positions with an "x"
+#'
+#' @param sequence Protein sequence
+#' @param positions_start Start position of the identified peptide.
+#' @param positions_end End position of the identified peptide.
+#'
+#' @return A modified protein sequence with each identified position replaced by "x"
+#' @import dplyr
+#' @import purrr
+#' @import tidyr
+#' @importFrom magrittr %>%
+#' @importFrom stringr str_sub
+#'
+#' @examples
+#' \dontrun{
+#' replace_identified_by_x(
+#' sequence = protein_sequence,
+#' positions_start = start,
+#' positions_end = end
```

Comment:
```suggestion
#' sequence = c("AEFGPEEAAVS"),
#' positions_start = c(1, 5, 8),
#' positions_end = c(3, 6, 11)
```
could be good to use this in order to show that the start and end can be vectorised while the protein sequence can't be. The sequence is still called a vector (but of length 1) since scalars do not exist in R.